<launch>
    <!-- Remapping image topics -->
    <!--remap from="rgb/image_rect" to="$(arg topic_rgb_image_rect)"/>
    <remap from="left/image_rect" to="$(arg topic_left_image_rect)"/>
    <remap from="right/image_rect" to="$(arg topic_right_image_rect)"/>
    <remap from="left/camera_info" to="$(arg topic_left_camera_info)"/>
    <remap from="right/camera_info" to="$(arg topic_right_camera_info)"/-->
 

    <!-- Parameters of loop closure transform nodes -->
    <!--param name="netvlad_distance" type="double" value="$(arg netvlad_distance)" />
    <param name="netvlad_dimensions" type="int" value="$(arg netvlad_dimensions)" />
    <param name="netvlad_batch_size" type="int" value="$(arg netvlad_batch_size)" />
    <param name="netvlad_max_matches_nb" type="int" value="$(arg netvlad_max_matches_nb)" />
    <param name="separators_min_inliers" type="int" value="$(arg separators_min_inliers)" />
    <param name="number_of_kf_skipped" type="int" value="$(arg number_of_kf_skipped)" /-->


    <!-- Nodes to run -->
    <!--node pkg="loop_closure_transform" type="stereo_cam_geometric_tools" name="stereo_cam_geometric_tools" output="screen" /-->
    <!--node pkg="loop_closure_transform" type="find_separators.py" name="find_separators" output="screen"/-->
    <!--node pkg="loop_closure_transform" type="factor_graph" name="factor_graph"/-->
    <node pkg="loop_closure_transform" type="loop_closure_transform_node" name="loop_closure_transform" output="screen"/>
</launch>